examples/panda_reacher.py

```diff
@@ -16,17 +19,14 @@ def main():
     ob = env.reset()
     print(f"Initial observation : {ob}")
     if goal:
-        from examples.scene_objects.goal import dynamicGoal
         env.add_goal(dynamicGoal)
 
     if obstacles:
-        from examples.scene_objects.obstacles import dynamicSphereObst2
         env.add_goal(dynamicGoal)
 
     if obstacles:
-        from examples.scene_objects.obstacles import dynamicSphereObst2
-
         env.add_obstacle(dynamicSphereObst2)
+
```

Comment:
I know about the drawbacks of in-file imports. However, motion planning scenes and thus all the obstacles and goals are an optional dependency. So either we make it a required dependency or we keep the in-file imports.